src/Shared/TrimmingAttributes.cs

```diff
@@ -119,3 +119,117 @@ public UnconditionalSuppressMessageAttribute(string category, string checkId)
     /// </summary>
     public string? Justification { get; set; }
 }
+
+[AttributeUsage(
```

Comment:
What's this for? NS2.0 projects?

Reply:
Yup. Healthchecks cross-compiles and needs it. The current approach is to include an internal attribute rather than ifdef the attribute usage.